setup.cfg

```diff
@@ -47,7 +47,7 @@ install_requires =
     iso-639
     isodate
     oyaml
-    PyJWT
+    PyJWT~=1.7
```

Comment:
CI seemed to be installing PyJWT==2.1.0

Reply:
Not okay to pin this, it's breaking existing installations. We should deal with the changes, I've already done that in gemma-zds-client as well so you can look at that for inspiration.